src/assets/redux/reducers.js

```diff
@@ -0,0 +1,33 @@
+import { FETCH_BOOKS, LOADING_FETCH_BOOKS, ERROR_FETCH_BOOKS } from "./actions";
+const defaultStore = {
+  loadingData: false,
+  books: [],
+  errorMsg: "",
+};
+
+export const reducerBooks = (state = defaultStore, action) => {
+  switch (action.type) {
+    case FETCH_BOOKS:
+      return {
+        loadingData: false,
+        books: action.books,
+        errorMsg: "",
+      };
+    case LOADING_FETCH_BOOKS:
+      return {
+        loadingData: true,
+        books: [],
```

Comment:
`books`, `errorMsg` przekazujesz w dwóch akcjach, po co? Skoro akcja odpowiada za loader to zmieniaj tylko stan loadera. Stan w reduxie jest niemutowalny tzn. powinienes najpierw zwracać aktualny state, a póżniej podmieniac te wartości, które Cię interesują:
` case LOADING_FETCH_BOOKS:
      return {
       ...state,
        isLoadingData: true,
      };` - do zmiany we wszystkich miejscach!

Dobrą praktyką jest nazywanie zmiennych `boolean` od `is...`